packages/salesforcedx-vscode-core/src/commands/forceApexClassCreate.ts

```diff
@@ -80,16 +80,24 @@ class SelectDirPath implements ParametersGatherer<{ outputdir: string }> {
     const rootPath = vscode.workspace.rootPath;
     let outputdir;
     if (rootPath) {
-      outputdir = this.explorerDir
-        ? path.relative(rootPath, this.explorerDir)
-        : await vscode.window.showQuickPick(
-            this.globDirs(rootPath, 'classes'),
-            <vscode.QuickPickOptions>{
-              placeHolder: nls.localize(
-                'force_apex_class_create_enter_dir_name'
-              )
-            }
-          );
+      const normalizedRoot = path.normalize(rootPath);
+      if (this.explorerDir) {
+        if (process.platform === 'win32') {
+          outputdir = path.normalize(this.explorerDir.substr(1));
```

Comment:
On windows, the explorerDir passed by the context menu has an extra '\' in the absolute path which breaks the relative path solution before.